tests/Token/TokenProviderTest.php

```diff
@@ -121,48 +124,6 @@ public function testCreatesFromCache()
         $this->assertEquals($token->getExpiration(), $found->getExpiration());
     }
 
-    public function testRefreshesFromCache()
```

Comment:
Why are we removing this?

Reply:
It was an accidentally added test.  It's actually tested below in testTokenProviderCachedSuccess.  This test wasn't even using a cache, and doing nothing